src/hooks/useGetImages.js

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useDispatch, useSelector } from "react-redux";
+import { setLoadmore } from "redux/actions";
+
+export default function useGetImages() {
+  const [images, setImages] = useState([]);
+  const { loadMore } = useSelector((state) => state);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (loadMore) {
+      // default api
+      let API = "http://localhost:4000/pics";
```

Comment:
If you were to use a "real" backend in production, how would you manage this variable?